src/input_reader.rs

```diff
@@ -258,9 +265,30 @@ impl Default for ActionSources {
     }
 }
 
-/// Tracks all consumed input from Bevy resources.
+/// All consumed input by actions in each schedule.
+#[derive(Resource, Default, Deref, DerefMut)]
+pub(crate) struct ConsumedInputs(TypeIdMap<IgnoredInputs>);
+
+/// Inputs that will be ignored until they return zero.
+///
+/// Once the input becomes zero, it will be automatically removed and no longer ignored.
```

Comment:
It would be nice to document (or link to documentation) about what "zero" means for the various kinds of input. Or choose a different term?